load-testing/app/scenarios/commonwealth-api-threads.yml

```diff
@@ -0,0 +1,122 @@
+config:
+  plugins:
+    metrics-by-endpoint:
+      useOnlyRequestNames: true
+      metricsNamespace: "latency_metrics"
+  environments:
+    us:
+      target: "https://us.unique.rocks"
+    eu:
+      target: "https://eu.unique.rocks"
+    local:
+      target: "http://localhost:8080"
+    beta:
+      target: "https://commonwealth-beta.herokuapp.com"
+    frick:
+      target: "https://commonwealth-frick.herokuapp.com"
+    frack:
+      target: "https://commonwealth-frack.herokuapp.com"
+  http:
+    extendedMetrics: true
+  phases:
+    - name: "Warm up"
+      duration: 10
+      arrivalRate: 5
+scenarios:
+  - name: "Read Endpoints"
+    flow:
+      # - get:
+      #     url: "/"
+      # - get:
+      #     url: "/api/domain"
+      # - get:
+      #     url: "/api/status"
+      # - post:
+      #     url: "/api/viewDiscussionNotifications"
+      #     form:
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - post:
+      #     url: "/api/viewChainEventNotifications"
+      #     form:
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - post:
+      #     url: "/api/getSubscribedChains"
+      #     form:
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - get:
+      #     url: "/api/viewSubscriptions"
+      #     qs:
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - get:
+      #     url: "/cmn-protocol"
+      # - get:
+      #     url: "/api/bulkOffchain"
+      #     qs:
+      #       chain: "cmn-protocol"
+      #       community:
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - post:
+      #     url: "/api/selectChain"
+      #     form:
+      #       chain: "cmn-protocol"
+      #       auth: true
+      #       jwt: "{{$processEnvironment.JWT}}"
+      # - post:
+      #     url: "/api/threadsUsersCountAndAvatars"
+      #     json:
+      #       chain: "cmn-protocol"
+      #       threads: []
+      # - get:
+      #     url: "/api/bulkTopics"
+      #     qs:
+      #       chain: "cmn-protocol"
+      - get:
+          name: "/api/activeThreads"
```

Comment:
I don't think this route exists anymore. Looks like this file needs to be heavily refactored since Ryan refactored routes.

Reply:
Thats just a name to report metrics against, API it calls is refactored one by Ryan
if we use POST /thread we it will merges numbers for BulkThreads, and GetThread, ActiveThread as they are behind same endpoint

This file is functional and tested against all endpoints listed in file.
When you run this it will print out activeThread ids etc

Reply:
i understand you are not a fan of these names, but thats what we still use in Controller

<img width="503" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/4791635/d7af992e-e603-41f2-9694-9d7587a43b3d">

These name do still have some context on backend
`get_bulk_threads`
`get_active_threads`

Reply:
I see - makes sense.